docker/scripts/testAll.sh

```diff
@@ -129,7 +190,11 @@ runTest query.sh 12 2
 #runTest mapReduceQuery.sh 12 2
 #runTest oozieQuery.sh 0 0
 
-$SCRIPT_DIR/cleanup.sh
+# Gives option to skip the cleanup stage
+if [ "${1}" == "-noCleanup" ] ; then
+    printTestSummary
+    exit 0
+fi
 
-echo
-echo "All tests SUCCEEDED!"
+printTestSummary
+$SCRIPT_DIR/cleanup.sh
```

Comment:
We should only cleanup all of the evidence if everything passed.  Otherwise we need a way to go and see what happened.

Reply:
I agree. Good idea. The latest commit adds a rework of the cleanup script to only delete the logs of tests that have passed. Logs related to tests that fail are kept.

Reply:
Could even remove `cleanup.sh` and put the cleanup in `testAll.sh` since that is the only script that uses the cleanup.